### PR TITLE
minor reformat of helpers

### PR DESCRIPTION
**Pull request by jeronimoek**

### CODE REVIEW

File: src/helpers/index.ts [hunk 1]
```diff
@@ -42,9 +37,7 @@ export const getBase255Number = (color: string, alpha = false): number => {
     }
     if (HEX.test(color)) {
         if (color.length === 3) {
-            return alpha
-                ? parseInt(color + color.slice(-1)) / 255
-                : parseInt(color + color.slice(-1));
+            color += color.slice(-1);
```
Comment (jeronimoek) [inline src/helpers/index.ts:40]: Here it's not just a reformat, now it also takes into account the MAX_DECIMALS value

File: src/helpers/index.ts [hunk 2]
```diff
@@ -21,18 +21,13 @@ export const getDEC = (hex: string): number => {
 
 //---Convert to hexadecimal
 export const getHEX = (number: NumberOrString): string => {
-    const hex = round(number).toString(16).toUpperCase();
-    if (hex.length === 1) { return `0x0${hex}`; }
+    const hex = round(number).toString(16).toUpperCase().padStart(2,'0');
```
Comment (elchininet) [inline src/helpers/index.ts:24]: The introduction of `padStart` is a breaking change for those using old `Node` versions. This would require to set a minimum `Node` version in the package and release it in a major version, but it doesn‘t worth it because the change is just aesthetic.

File: src/helpers/index.ts [hunk 3]
```diff
@@ -55,7 +48,7 @@ export const getBase255Number = (color: string, alpha = false): number => {
 
 
 //---Calculate a decimal 0-1 value from CMYK value
-export const getCMYKNumber = (color: string): number => Math.min(PCENT.test(color) ? +(color.replace(PCENT, '$1')) / 100 : +color, 1);
+export const getCMYKNumber = (color: string): number => Math.min(PCENT.test(color) ? percent(color) / 100 : +color, 1);
```
Comment (elchininet) [inline src/helpers/index.ts:51]: The `pcent` utility takes care of the first part of the code I think. Maybe it could be shortened in:

```
export const getCMYKNumber = (color: string): number => Math.min(percent(color) / 100, 1);
``` 

Test it on your side to see if it covers all the edge cases.

Comment (jeronimoek) [reply src/helpers/index.ts:51]: I tested it and using the code you mentioned breaks the tests, the thing is that the "percent" function just translates a percentage string into a number, it doesn't divide it by 100

**Comment by jeronimoek** [comment]
fixed

File: src/helpers/index.ts (hunk 3 above)
Comment (elchininet) [reply src/helpers/index.ts:56]: Yes I got it. It is just that it checks for `PCENT.test(color)` and executes a function that checks again for the same. I would rather separate the logic of `+(color.replace(PCENT, '$1'))` into a separated utility function and use it in:

* helpers `percent`
* helpers `getBase255Number`
* helpers `getCMYKNumber`
* utils `normalizeAlpha`

Comment (jeronimoek) [reply src/helpers/index.ts]: fixed as requested

**Review by elchininet — APPROVED** [review]
